src/main.rs

```diff
@@ -477,7 +479,13 @@ async fn get_eth_head(State(state): State<Arc<AppState>>) -> impl IntoResponse {
                             let now = Utc::now().timestamp() as u64;
                             (
                                 StatusCode::OK,
-                                [("Cache-Control", "public, max-age=7200, must-revalidate")],
+                                [(
+                                    "Cache-Control",
+                                    format!(
+                                        "public, max-age={}, must-revalidate",
+                                        state.eth_head_response_cache
+                                    ),
```

Comment:
Why it didn't work before? 🤔

Reply:
different types were returned, in this case, it is `&str` and `String` and in the other place was `&str` and `&str`